debian/control

```diff
@@ -0,0 +1,18 @@
+Source: nnstreamer-ros
+Section: libs
+Priority: optional
+Maintainer: MyungJoo Ham <myungjoo.ham@samsung.com>
+Build-Depends: gcc (>=5), cmake, debhelper (>=9),
+ libgstreamer1.0-dev, libgstreamer-plugins-base1.0-dev, libglib2.0-dev,
+ gstreamer1.0-tools, libgtest-dev, ssat, python2.7, nnstreamer-dev,
+ ros-kinetic-catkin, ros-kinetic-cmake-modules,
+ ros-kinetic-genmsg, ros-kinetic-message-generation,
+ ros-kinetic-roscpp
+Standards-Version: 3.9.7
```

Comment:
Should we change the version number?

Reply:
@again4you This version is for the debian standards, not for our project.